src/Directory.Packages.props

```diff
@@ -9,7 +9,8 @@
     <PackageVersion Include="Uno.Cupertino" Version="2.4.1" />
     <PackageVersion Include="Uno.Cupertino.WinUI" Version="2.4.1" />
     <PackageVersion Include="Uno.Extensions.Markup.Generators" Version="1.0.0-dev.125" />
-    <PackageVersion Include="Uno.WinUI.Markup" Version="4.6.0-dev.12" />
+    <PackageVersion Include="Uno.Themes.WinUI.Markup" Version="2.5.0-dev.3" />
+    <PackageVersion Include="Uno.WinUI.Markup" Version="4.6.0-dev.14" />
```

Comment:
these will need to be updated due to breaking changes in Uno.Extensions.Markup. See unoplatform/Uno.Themes/pull/906

Reply:
let's be sure to update these dependencies